packages/core/test/util.js

```diff
@@ -228,11 +228,20 @@ function createScenarioFromScaffold ({
   }
 }
 
+function isSpam(message){
+  if(typeof message === 'string' && message.includes('Removing intrinsics.')){
+    return true
+  }
+}
+
 function createHookedConsole () {
   let hasResolved = false
   let resolve
   const firstLogEventPromise = new Promise(_resolve => { resolve = _resolve })
   const hookedLog = (message) => {
+    if (isSpam(message)) {
+      return
+    }
```

Comment:
workaround for SES printing `Removing intrinsics.Object.hasOwn` in Node 16. ignore it;)